packages/engines/package.json

```diff
@@ -8,7 +8,7 @@
   "devDependencies": {
     "@prisma/debug": "2.26.0",
     "@prisma/engines-version": "workspace:2.27.0-14.95b10778266ed1eb3013872ab5c09e460bd941fe",
-    "@prisma/fetch-engine": "2.26.0",
+    "@prisma/fetch-engine": "workspace:*",
```

Comment:
Is it weird that the new one uses `*` while the other one has a specific version?